package.json

```diff
@@ -28,11 +28,11 @@
     "webpack": "^1.12.12"
   },
   "scripts": {
-    "build": "npm run build:main && npm run build:fp",
+    "build": "npm run build:main & npm run build:fp",
     "build:fp": "node lib/fp/build-dist.js",
     "build:fp-modules": "node lib/fp/build-modules.js",
     "build:main": "node lib/main/build-dist.js",
-    "style": "npm run style:main && npm run style:fp && npm run style:perf && npm run style:test",
+    "style": "npm run style:main & npm run style:fp & npm run style:perf & npm run style:test",
```

Comment:
Never used jscs. But I'm guessing this will raise a flag about errors, then when you're figuring them out you can run each one individually...